requirements/requirements_ansible.in

```diff
@@ -43,7 +43,7 @@ azure-mgmt-iothub==0.7.0
 # AWS
 boto==2.47.0    # last which does not break ec2 scripts
 boto3==1.9.223
-jinja2==2.10.1  # required for native jinja2 types for inventory compat mode
```

Comment:
The requirements files relevant to the sending of notifications are `requirements.in` and `requirements.txt`; the `_ansible` files correspond to the ansible venv which only come into play when a playbook, ad hoc command, etc. are run. Notifications are executed within the awx venv.

It looks like while our current `requirements.in` doesn't specify a minimum version for jinja, the `requirements.txt` file specifies the pinned version as `2.11.12` (this would explain why your changes were able to work despite the requirements not being updated). Seems like it would be worth making the minimum required version explicit in `requirements.in`.